Library/Homebrew/brew.sh

```diff
@@ -365,6 +365,19 @@ then
   export HOMEBREW_BOTTLE_DOMAIN="$HOMEBREW_BOTTLE_DEFAULT_DOMAIN"
 fi
 
+HOMEBREW_DEFAULT_BREW_REMOTE="https://github.com/Homebrew/brew.git"
+if [[ -z "$HOMEBREW_BREW_REMOTE" ]]
+then
+  export HOMEBREW_BREW_REMOTE="$HOMEBREW_DEFAULT_BREW_REMOTE"
```

Comment:
```suggestion
 HOMEBREW_BREW_REMOTE="$HOMEBREW_DEFAULT_BREW_REMOTE"
```

Reply:
Add after https://github.com/Homebrew/brew/blob/e112110297be66f1ef6d396d80f2a7093643ef67/Library/Homebrew/brew.sh#L244

`export HOMEBREW_BREW_REMOTE` and `export HOMEBREW_DEFAULT_CORE_REMOTE`